helm/ingress.sh

```diff
@@ -4,12 +4,10 @@ set -x
 
 export KUBECONFIG=$1
 
-kubectl --namespace kube-system create sa tiller
-kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
-
-helm init --service-account tiller --wait --upgrade
-helm repo update
+helm init
+helm repo add nginx https://helm.nginx.com/stable
 
 # https://github.com/helm/charts/tree/master/stable/nginx-ingress
 # https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/configmap.md
-helm upgrade --install ingress stable/nginx-ingress --set controller.config.enable-underscores-in-headers='"true"',controller.config.use-forwarded-headers='"true"'
+
+helm install ingress-controller nginx/nginx-ingress --set controller.config.enable-underscores-in-headers='"true"',controller.config.use-forwarded-headers='"true"'
```

Comment:
Use the kubernetes/ nginx-ingress not the nginx version. Just use Kubernetes community services.

Reply:
Also use `helm upgrade --install` it allows us to upgrade an existing installation if it already exists.